### PR TITLE
Various updates to SimEng documentation

### PR DESCRIPTION
**Pull request by jj16791**
updates to the docs for release 0.9.5

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,15 +1,16 @@
-Welcome to the 0.9.4 open source release of the Simulation Engine! SimEng is a fast, easy to use and modify, cycle-level simulator for CPUs. Its initial focus is on simulating single Arm cores in server CPUs, and so the instruction set architecture (ISA) target is initially Armv9.2-a+SVE. Later versions of the Arm ISA, and other ISAs such as RISC-V, will be supported in future releases.
+Welcome to the 0.9.5 open source release of the Simulation Engine! SimEng is a fast, easy to use and modify, cycle-level simulator for CPUs. Its initial focus is on simulating single Arm cores in server CPUs, and so the instruction set architecture (ISA) target is initially Armv9.2-a+SVE. Later versions of the Arm ISA, and other ISAs such as RISC-V, will be supported in future releases.
```
Comment (FinnWilkinson) [inline RELEASE-NOTES.txt:1]: Arm9.2-a+SVE+SME*
Also we now support RISC-V (RV64) but this paragraph says we don't

File: docs/sphinx/user/configuring_simeng.rst [hunk 2]
```diff
@@ -76,23 +80,33 @@ Stack- Size
...
+- Predicate-Count (Optional)
     The number of physical Arm SVE extension predicate registers.
 
-Conditional-Count
+- Conditional-Count
     The number of physical status/flag/conditional-code registers.
 
-Matrix-Count
+- Matrix-Count (Optional)
     The number of physical ``za`` SME registers.
```
Comment (FinnWilkinson) [inline docs/sphinx/user/configuring_simeng.rst:100]: *Number of physical ``za`` Arm SME registers.

**Review by FinnWilkinson — CHANGES_REQUESTED** [review]

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (jj16791) [reply RELEASE-NOTES.txt:1]: Added some stuff for RISC-V

File: RELEASE-NOTES.txt [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
 
 You can find all the user and developer documentation on the SimEng webpage:
 
    https://uob-hpc.github.io/SimEng/index.html
 
-SimEng should be fast, much faster than other cycle-level CPU simulators. The fastest hardware we’ve run it on is a 2021 Apple M1 Mac mini, where we’ve measured SimEng running at up to 1.5 MIPS for the Out of Order (OoO) core in our ThunderX2 and A64fx models. SVE (vector) codes are more complex to simulate and tend to be slower - on the same hardware we saw simulated SVE codes running at around 0.6 MIPS. SimEng should also be accurate - most of the codes we’ve tested report cycle counts within 5-10% of real hardware, for our ThunderX2 and A64fx models.
+SimEng should be fast, much faster than other cycle-level CPU simulators. The fastest hardware we’ve run it on is a M1-Ultra mac studio, where we’ve measured SimEng running at up to ~1 MIPS for the Out of Order (OoO) core in our ThunderX2 and A64fx models. SVE (vector) codes are more complex to simulate and tend to be slower - on the same hardware we saw simulated SVE codes running at around 0.6 MIPS.
```
Comment (FinnWilkinson) [inline RELEASE-NOTES.txt:9]: *Mac Studio

**Review by FinnWilkinson — CHANGES_REQUESTED** [review]

**Review by FinnWilkinson — APPROVED** [review]

File: RELEASE-NOTES.txt [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
 
 You can find all the user and developer documentation on the SimEng webpage:
 
    https://uob-hpc.github.io/SimEng/index.html
 
-SimEng should be fast, much faster than other cycle-level CPU simulators. The fastest hardware we’ve run it on is a 2021 Apple M1 Mac mini, where we’ve measured SimEng running at up to 1.5 MIPS for the Out of Order (OoO) core in our ThunderX2 and A64fx models. SVE (vector) codes are more complex to simulate and tend to be slower - on the same hardware we saw simulated SVE codes running at around 0.6 MIPS. SimEng should also be accurate - most of the codes we’ve tested report cycle counts within 5-10% of real hardware, for our ThunderX2 and A64fx models.
+SimEng should be fast, much faster than other cycle-level CPU simulators. The fastest hardware we’ve run it on is a M1-Ultra Mac Studio, where we’ve measured SimEng running at up to ~1 MIPS for the Out of Order (OoO) core in our ThunderX2 and A64fx models. SVE (vector) codes are more complex to simulate and tend to be slower - on the same hardware we saw simulated SVE codes running at around 0.6 MIPS.
```
Comment (dANW34V3R) [inline RELEASE-NOTES.txt:9]: Is it "an M1-Ultra", not sure?

File: RELEASE-NOTES.txt [hunk 5]
```diff
@@ -22,12 +23,11 @@ Pre-rolled CPU models in SimEng include:
...
-* We only support a subset of Linux's system calls, and these run under emulation. The subset of 42 calls is enough to run all the codes we've tried so far
+* We only support a subset of Linux's system calls, and these run under emulation. The subset of 43 calls is enough to run all the codes we've tried so far
 * Only partial support for micro-oping, mainly for Load and Store operations
-* Only partial support for the ISA - we’ve focused on implementing the instructions we’ve seen generated by the compiler for all the test cases we’ve considered, about 930 different instructions so far (~15% of the entire ISA). You’re likely to come across “instruction not implemented” errors, just let us know when you see these
+* Only partial support for the ISA - we’ve focused on implementing the instructions we’ve seen generated by the compiler for all the test cases we’ve considered, about 1000 different AArch64 instructions (~15% of the entire ISA) and about 150 different RISC-V instructions so far (~33% of the rv64ima target). You’re likely to come across “instruction not implemented” errors, just let us know when you see these
```
Comment (dANW34V3R) [inline RELEASE-NOTES.txt:28]: Where did the 33% RISC-V number come from?

**Review by dANW34V3R — CHANGES_REQUESTED** [review]
Looks good on the whole. Just small queries

File: RELEASE-NOTES.txt (hunk 5 above)
Comment (jj16791) [reply RELEASE-NOTES.txt:28]: Forgot to change this, was meant to say (~33% of the RISC-V ISA supported by Capstone). That okay?

File: RELEASE-NOTES.txt (hunk 4 above)
Comment (jj16791) [reply RELEASE-NOTES.txt:9]: Think you're right, will change.

**Review by dANW34V3R — APPROVED** [review]
All looks good

**Pull request merged**